srw_detector.py

```diff
@@ -6,6 +6,7 @@
 from ophyd import Device, Signal, Component as Cpt
 from ophyd.sim import SynAxis, NullStatus, new_uid
 
+from srw_run import srw_run
```

Comment:
I removed this line in my master branch recently. Can you either remove it, or do `git pull --all` and `git merge upstream/master`? If you didn't configure the remote, configure it with `git remote add upstream https://github.com/mrakitin/sirepo_bluesky`. Then push the changes, which will update the code in the PR.

Reply:
Should be fixed now.